test/db/analysis/java

```diff
@@ -320,58 +327,58 @@ CMDS=javac
 EXPECT=<<EOF
 Version: (51.0) Java SE 1.7
 Flags: (0x0021) public super
-Class: (#18) JDWPExecCmd
+Class: (#18) LJDWPExecCmd;
 Super: (#19) java/lang/Object
 Constant pool: 71
-   #1 = Methodref           #19:#33        // java.lang.Object:<init>:()V
-   #2 = Class               #34            // java.lang.StringBuilder
-   #3 = Methodref           #2:#33         // java.lang.StringBuilder:<init>:()V
-   #4 = Methodref           #2:#35         // java.lang.StringBuilder:append:(Ljava.lang.String;)Ljava.lang.StringBuilder;
+   #1 = Methodref           #19:#33        // Ljava/lang/Object;.<init>()V
+   #2 = Class               #34            // Ljava/lang/StringBuilder;
+   #3 = Methodref           #2:#33         // Ljava/lang/StringBuilder;.<init>()V
+   #4 = Methodref           #2:#35         // Ljava/lang/StringBuilder;.append(Ljava/lang/String;)Ljava/lang/StringBuilder;
    #5 = String              #36            // " output:\n"
-   #6 = Methodref           #2:#37         // java.lang.StringBuilder:toString:()Ljava.lang.String;
-   #7 = Methodref           #38:#39        // java.lang.Runtime:getRuntime:()Ljava.lang.Runtime;
-   #8 = Methodref           #38:#40        // java.lang.Runtime:exec:(Ljava.lang.String;)Ljava.lang.Process;
-   #9 = Class               #41            // java.io.BufferedReader
-  #10 = Class               #42            // java.io.InputStreamReader
-  #11 = Methodref           #43:#44        // java.lang.Process:getInputStream:()Ljava.io.InputStream;
-  #12 = Methodref           #10:#45        // java.io.InputStreamReader:<init>:(Ljava.io.InputStream;)V
-  #13 = Methodref           #9:#46         // java.io.BufferedReader:<init>:(Ljava.io.Reader;)V
-  #14 = Methodref           #9:#47         // java.io.BufferedReader:readLine:()Ljava.lang.String;
-  #15 = Methodref           #48:#49        // java.lang.String:trim:()Ljava.lang.String;
+   #6 = Methodref           #2:#37         // Ljava/lang/StringBuilder;.toString()Ljava/lang/String;
```

Comment:
Is it possible to demangle these comments as well? I think the previous output looked better than the `L../../` strings.

Reply:
yes.

Reply:
done.